### ISSUE TITLE
Hey

### ISSUE THREAD

**Issue by Serkules** [frogstack/HSTeamPlay#1]
I tried to install it as you said but didnt get it to work. I got this message below.

If you can help it would be appreciated. Contact me on google hangouts serkules5@gmail.com

![screenshot_1](https://cloud.githubusercontent.com/assets/22824093/25777052/5a73da42-32d1-11e7-8d9b-824241c2a4db.png)

**Comment by Serkules** [comment]
Could you help to make it work ?

**Comment by andrewanderson** [comment]
Where did you put your source? And where are you building it from?

**Comment by Serkules** [comment]
I put it on the desktop and im building it from the folder HSteamplay

C:\Users\XXX\Desktop

Could you please help to get it working ?

**Comment by Serkules** [comment]
@andrewanderson Could you help ?

**Comment by varandru** [comment]
I was getting the same problem. This part was solved by moving the whole HSTeamPlay folder to $GOROOT, in my case C:\Go\src\ . However, I get another set of errors: 

    hearthstone\cardRater.go:6:2: cannot find package "github.com/ChrisHines/GoSkills/skills" in any of:
            C:\Go\src\vendor\github.com\ChrisHines\GoSkills\skills (vendor tree)
            C:\Go\src\github.com\ChrisHines\GoSkills\skills (from $GOROOT)
            C:\Users\HP\go\src\github.com\ChrisHines\GoSkills\skills (from $GOPATH)
    hearthstone\cardRater.go:7:2: cannot find package "github.com/ChrisHines/GoSkills/skills/trueskill" in any of:
            C:\Go\src\vendor\github.com\ChrisHines\GoSkills\skills\trueskill (vendor tree)
            C:\Go\src\github.com\ChrisHines\GoSkills\skills\trueskill (from $GOROOT)
            C:\Users\HP\go\src\github.com\ChrisHines\GoSkills\skills\trueskill (from $GOPATH)

Quick google search shows that vendoring doesn't work, but I haven't used anything related to GO until half an hour ago, so I have no idea what vendoring is and how I am supposed to fix it. Will keep looking, though.

**Comment by varandru** [comment]
Got it. If you have GitHub working from command line, `go get github.com/ChrisHines/GoSkills/skills` should help. If you don't, download the library and extract it to ' %GOPATH%/github.com/ChrisHines/GoSkills/skills '

**Comment by frogstack** [comment]
The above two issues have been fixed.
- Putting the source in $GOROOT was bad form. @andrewanderson fixed that in https://github.com/frogstack/HSTeamPlay/commit/adb12d7635bed1971533cf3d36f60972d0b49f17
- I included a copy of TrueSkill, not knowing that it was not being used. I've removed it and updated the readme to indicate the necessary `go get` command in https://github.com/frogstack/HSTeamPlay/commit/c86c7b738ef6af8cb3acf3898bed5997ee72883c
